calico/felix/endpoint.py

```diff
@@ -585,7 +604,7 @@ def _on_worker_died(self, watch_greenlet):
 
 class LocalEndpoint(RefCountedActor):
 
-    def __init__(self, config, combined_id, ip_type, iptables_updater,
+    def __init__(self, config, combined_id, ip_type, iptables_updater, untracked_updater,
```

Comment:
If this code was going to stay around, I'd advise creating a an `__init__` method in the two  subclasses so you don't end up passing `None` for this argument when creating a `WorkloadEndpoint` but it's only a minor issue so probably not worth the effort.